### DISCUSSION TITLE
Can I repost this to Nexus Mods?

### DISCUSSION THREAD

**Discussion by Loloman132** [leoetlino/project-restoration#245] [Q&A]
I have been looking for mm3d mods on nexus mods for a long time, but now I want to do somthing about it.in nexus mods, you can suggest a game, however, there needs to be a working mod file for them to consider it.

**Comment by leoetlino** [comment]
You are legally allowed to do so as long as you provide proper attribution. But honestly I don't really see the point considering that Nexus Mods is mostly for PC games.
